themes/red-theme/lms/templates/footer.html

```diff
@@ -0,0 +1,111 @@
+## mako
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from microsite_configuration.templatetags.microsite import platform_name
+%>
+
+<div class="wrapper wrapper-footer">
+  <footer>
+    <!-- This is super-ugly, don't use it! -->
```

Comment:
I wrote a unit test for the theming that looks for specific text to see that the template lookup is happening properly, and "super-ugly" is the text it is looking for.  I'm happy to choose something different.  If we make this theme more straight-laced, we can scrap this comment and use something else.